starter-code/index.html

```diff
@@ -1,42 +1,79 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
   <title>Spotify</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
+<nav class = "spotify-nav">
+    <img src ="./images/spotify-logo.png" width="250 vh">
+  <ul class = "spotify-nav-ul">
+    <li><a href="#">Download</a></li>
+    <li><a href="#">Discover</a></li>
+    <li><a href="#">Help</a></li>
+    <li><a href="#">Premium</a></li>
+  </ul>
+</nav>
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
 
-What’s on Spotify?
+<header class = "header-spotify">
+ <h1>Music for everyone.</h1>
+    <p>Spotify is now free on mobile, tablet and computer.<br>
+    Listen to the right music, wherever you are.</p>
+</header>
 
-Millions of Songs
-There are millions of
-songs on Spotify
 
-HD Music
-Listen to music as if you
-were listening live
+<section class= "about-spotify">
+<h2>What’s on Spotify?
+</h2>
+<div class = "col col-3">
+    <img src ="./images/music-icon.png" width="130 vh">
+  <h3>Millions of Songs</h3>
+    <p>There are millions of<br>songs on Spotify</p>
+</div>
+<div class = "col col-3">
+    <img src ="./images/high-quality-icon.png" width="150 vh">
+  <h3>HD Music</h3>
+    <p>Listen to music as if you <br> were listening live</p>
+</div>
+<div class = "col col-3">
+    <img src ="./images/devices-icon.png" width="150 vh">
+  <h3>Stream Everywhere</h3>
+    <p>Stream music on your<br>
+    smartphone, tablet or<br>
+    computer</p>
+    <div class="clearfix"></div>
+</div>
+</section>
 
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
 
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<section class = "kanye-west">
+<h2>It’s as yeezy as Kanye West.</h2>
+  <ul>
+    <li>
+      <h3>Seach</h3>
+      <p>Know what you want to listen to?<br>
+          Just search and hit play.</p>
+    </li>
+    <li>
+      <h3>Browse</h3>
+      <p>Check out the latest charts,<br>
+          brand new releases and great<br>
+          playlists for right now.</p>
+    </li>
+    <li>
+      <h3>Discover</h3>
+      <p>Enjoy new music every Monday<br>
+          with your own personal playlist.<br>
+          Or sit back and enjoy Radio.</p>
+    </li>
+  </ul>
+  <div class = "white-logo"><img src="./images/spotify-icon-white.png" width="150 vh"></div>
+  <div class="app-kanye-west"><img src="./images/spotify-app.jpg" width="250 vh"></div>
```

Comment:
This maybe would have been more manageable if you had split it in 3 parts giving each one a third of the space of the parent. Or in two giving more to one of the parts